doc/reference/reference_lua/net_box.rst

```diff
@@ -132,9 +132,7 @@ Below is a list of all ``net.box`` functions.
         *   -   :ref:`conn:eval() <net_box-eval>`                                
             -   Evaluate the expression in a string and execute it                
         *   -   :ref:`conn:call() <net_box-call>`                      
-            -   Call a stored procedure               
-        *   -   :ref:`conn:timeout() <conn-timeout>`                               
```

Comment:
What happened to `conn:timeout()`?

Reply:
I've checked - looks like it was removed from the API